cmd/filegen/main.go

```diff
@@ -97,13 +107,23 @@ VERSION:
 	initialNodesPkPlainFileName    = "./initialNodesPkPlain.txt"
 	genesisFilename                = "./genesis.json"
 	nodesSetupFilename             = "./nodesSetup.json"
+	txgenAccountsFileName          = "./accounts.json"
 
 	errInvalidNumPrivPubKeys = errors.New("invalid number of private/public keys to generate")
 	errInvalidMintValue      = errors.New("invalid mint value for generated public keys")
 	errInvalidNumOfNodes     = errors.New("invalid number of nodes in shard/metachain or in the consensus group")
 	errCreatingKeygen        = errors.New("cannot create key gen")
 )
 
+type txgenAccount struct {
+	PubKey        string   `json:"pubKey"`
+	PrivKey       string   `json:"privKey"`
+	LastNonce     uint64   `json:"lastNonce"`
+	Balance       *big.Int `json:"balance"`
+	TokenBalance  *big.Int `json:"tokenBalance"`
```

Comment:
what is the difference between balance and token balance?

Reply:
token balance represents the balance inside an ERC20 smart contract